lib/data/dir/matrix.json

```diff
@@ -1,10 +1,13 @@
 {
   "stakeholders": [
```

Comment:
Please work to change all of the content to either be gender-neutral (they) or change it to include both genders (he/she, him/her, etc). Over 40% of DV victims are male, so I think it's important that we include anyone who may be a perpetrator and/or victim.